### PR TITLE
Listen and print backend logs after starting debug session

### PR DESCRIPTION
**Pull request by ykandrashou**

### CODE REVIEW

File: src/commands/veloce/debug/logs.ts [hunk 1]
```diff
@@ -56,15 +56,19 @@ export default class Org extends SfdxCommand {
...
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      try {
+        const response = await axios.get(`${backendUrl}/services/dev-override/logs`, {headers});
+        if (response.data !== '') {
+          this.ux.log(response.data)
+        }
+      } catch (e: any) {
+        this.ux.log(`Failed to get logs: ${e as string}`)
```
Comment (veloceapps) [inline src/commands/veloce/debug/logs.ts:69]: tut nuzhno toString() ?
as nuzhno kak mozhno menishe ispolizovatj

File: src/commands/veloce/debug/logs.ts [hunk 2]
```diff
@@ -56,15 +56,19 @@ export default class Org extends SfdxCommand {
...
+    this.ux.log('Following the backend logs...')
+
+    // eslint-disable-next-line no-constant-condition
+    while (true) {
+      try {
+        const response = await axios.get(`${backendUrl}/services/dev-override/logs`, {headers});
+        if (response.data !== '') {
+          this.ux.log(response.data)
+        }
+      } catch (e: any) {
```
Comment (veloceapps) [inline src/commands/veloce/debug/logs.ts:68]: tut any navernoe mozhno izbezhatj? estj error tip?

File: src/commands/veloce/debug/logs.ts [hunk 3]
```diff
@@ -56,15 +56,19 @@ export default class Org extends SfdxCommand {
...
-    try {
-      await axios.post(`${backendUrl}/services/dev-override/logs`, {}, {
-        headers
-      })
-    } catch ({ data }) {
-      this.ux.log(`Failed to list debug sessions: ${data as string}`)
-      return {}
+    this.ux.log('Following the backend logs...')
+
+    // eslint-disable-next-line no-constant-condition
```
Comment (veloceapps) [inline src/commands/veloce/debug/logs.ts:61]: nikak nelizja bez otkluchenija lintera?

File: src/commands/veloce/debug/logs.ts (hunk 2 above)
Comment (ykandrashou) [reply src/commands/veloce/debug/logs.ts]: Fixed.

File: src/commands/veloce/debug/logs.ts (hunk 1 above)
Comment (veloceapps) [reply src/commands/veloce/debug/logs.ts]: this was moved just to have "better start" not as a source if ideal code. we are trying to write better code here

File: src/commands/veloce/debug/logs.ts [hunk 4]
```diff
@@ -51,20 +51,28 @@ export default class Org extends SfdxCommand {
...
-      })
-    } catch ({ data }) {
-      this.ux.log(`Failed to list debug sessions: ${data as string}`)
-      return {}
+      const response = await axios.get(`${backendUrl}/services/dev-override/logs`, {headers});
+      if (response.data !== '') {
+        this.ux.log(response.data)
+      }
+    } catch (e) {
+      this.ux.log(`Failed to get logs: ${e as string}`)
```
Comment (veloceapps) [inline src/commands/veloce/debug/logs.ts:73]: this example:
 catch (error) {
          const err = error as AxiosError
          this.ux.log(`Failed to deploy ${group.name}: ${err?.response?.data} code ${err?.response?.status}`);
        }
lets agree on catches to not be just "as string".

Comment (ykandrashou) [reply src/commands/veloce/debug/logs.ts]: I applied your suggestion. Thanks.

**Pull request merged**